cql3/query_processor.cc

```diff
@@ -95,14 +95,14 @@ query_processor::query_processor(service::storage_proxy& proxy, data_dictionary:
     qp_group.push_back(sm::make_counter(
         "statements_prepared",
         _stats.prepare_invocations,
-        sm::description("Counts the total number of parsed CQL requests.")));
+        sm::description("Counts the total number of parsed CQL requests.")).set_skip_when_empty());
```

Comment:
How would this ever be empty?

Reply:
If there are no cql requests

Reply:
There are always internal requests during boot. But maybe not on all shards.

Reply:
I've removed that line from the commit. It will not have a huge impact any how